src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApp.scala

```diff
@@ -1,25 +1,118 @@
 package org.broadinstitute.dsde.firecloud
 
-import scala.concurrent.duration._
-import akka.actor.{ActorSystem, Props}
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.stream.ActorMaterializer
 import com.typesafe.scalalogging.LazyLogging
-import org.broadinstitute.dsde.vault.common.util.ServerInitializer
+import org.broadinstitute.dsde.firecloud.dataaccess._
+import org.broadinstitute.dsde.firecloud.elastic.ElasticUtils
+import org.broadinstitute.dsde.firecloud.metrics.MetricsActor
+import org.broadinstitute.dsde.firecloud.model.{ModelSchema, UserInfo, WithAccessToken}
+import org.broadinstitute.dsde.firecloud.service._
+import org.broadinstitute.dsde.workbench.util.health.HealthMonitor
+import org.elasticsearch.client.transport.TransportClient
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
 
-object FireCloudApp extends LazyLogging {
+object FireCloudApp extends App with LazyLogging {
```

Comment:
this is analogous to `Boot.scala` in rawls. maybe i'll rename it so we have some consistency? TBD

Reply:
I like the consistency of similar naming, but it's not super duper important